python/cudf/cudf/tests/test_orc.py

```diff
@@ -604,13 +604,13 @@ def normalized_equals(value1, value2):
 
 
 @pytest.mark.parametrize("stats_freq", ["STRIPE", "ROWGROUP"])
-@pytest.mark.parametrize("nrows", [1, 100, 6000000])
+@pytest.mark.parametrize("nrows", [1, 100, 100000])
 def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
     from pyarrow import orc
 
     supported_stat_types = supported_numpy_dtypes + ["str"]
     # Can't write random bool columns until issue #6763 is fixed
-    if nrows == 6000000:
+    if nrows == 100000:
```

Comment:
Why does this work for `nrows=100`?

Reply:
We can write a single row group of random bools, just not multiple (at least not in the way that does not cause issues with **other** readers). So anything below 10k rows is fine. I know this is very hacky :(